.github/workflows/repo-metrics-weekly.yml

```diff
@@ -0,0 +1,58 @@
+name: Weekly repo metrics
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 9 * * MON'
+
+permissions:
+  issues: write
+  pull-requests: read
+
+jobs:
+  build:
+    # this workflow will always fail in forks; bail if this isn't running in the upstream
+    if: github.repository == 'aws/aws-cdk'
+    name: metrics
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Get dates for last week
+      shell: bash
+      run: |
+        # Calculate the date of the previous Monday
+        PREVIOUS_MONDAY=$(date -d "7 days ago" "+%Y-%m-%d")
+
+        # Calculate the date of the current Sunday
+        CURRENT_SUNDAY=$(date -d "1 day ago" "+%Y-%m-%d")
+
+        # Set an environment variable with the date range
+        echo "$PREVIOUS_MONDAY..$CURRENT_SUNDAY"
+        echo "last_week=$PREVIOUS_MONDAY..$CURRENT_SUNDAY" >> "$GITHUB_ENV"
+
+    - name: Report on issues
+      uses: github/issue-metrics@v2
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SEARCH_QUERY: 'repo:aws/aws-cdk is:issue created:${{ env.last_week }} -reason:"not planned"'
+
+    - name: Create report for issues
+      uses: peter-evans/create-issue-from-file@v5
+      with:
+        title: Weekly issue metrics report
+        token: ${{ secrets.GITHUB_TOKEN }}
+        content-filepath: ./issue_metrics.md
+        assignees: paulhcsun
+        
+    - name: Report on PRs
+      uses: github/issue-metrics@v2
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SEARCH_QUERY: 'repo:aws/aws-cdk is:pr created:${{ env.last_week }} -is:draft'
+
+    - name: Create report for PRs
+      uses: peter-evans/create-issue-from-file@v5
```

Comment:
Not a blocker. Would like to know is it normal and safe to use third party github actions for aws owned repos? As I can see this action is not from a `Verified creator` label which usually GITHUB does.

peter-evans/create-issue-from-file@v5 -> https://github.com/marketplace/actions/create-issue-from-file

Sample verified action: https://github.com/marketplace/actions/checkout
<img width="274" alt="image" src="https://github.com/aws/aws-cdk/assets/4015201/6a20213e-6f4f-4d02-94c2-bc965e5544be">